src/schema/v2/conversation/index.ts

```diff
@@ -303,10 +304,22 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    partnerCollectorProfile: {
+      description:
+        "The collector profile of the user who initiated the conversation. Do not use this field for Partners",
+      type: PartnerCollectorProfileType,
+      resolve: async ({ from_id, to_id }) => {
+        return {
+          partnerId: to_id,
+          collectorId: from_id,
+        }
+      },
+    },
     fromProfile: {
       description:
         "The collector profile of the user who initiated the conversation",
       type: CollectorProfileType,
+      deprecationReason: "Use `partnerCollectorProfile` instead",
```

Comment:
**Question:** We used to have:

`fromUser` - deprecated. "It will be inaccessible to partners in future versions. Prefer using `fromProfile` instead."

`fromProfile` - it is becoming deprecated in favor of `partnerCollectorProfile`.

Should we address the previous deprecations?